### DISCUSSION TITLE
Sign up error

### DISCUSSION THREAD

**Discussion by adepauldami** [AntlerVC/firetable#982] [Support / Q&A] [answered]
I've not been able to create a rowy account despite the fact I used different methods

**Comment by RajGM** [comment]
Hi @adepauldami 
Can you please share the screenshot of the error?

**Comment by BeeBombshell** [comment, marked as answer]
Hey @adepauldami, when you try to sign up, does it open the auth window? Also, does this happen for all three methods?

**Reply by BeeBombshell** [reply]
@adepauldami has this issue been resolved?

**Reply by BeeBombshell** [reply]
Hey guys, closing this discussion since it has been unattended for a long time. @adepauldami if you're still facing this issue, feel free to create a new one.
